src/System.Console/src/System/Console.cs

```diff
@@ -728,31 +728,34 @@ public static void Write(String value)
             Out.Write(value);
         }
 
-        // TODO: Uncomment when ArgIterator and friends are available in System.Runtime.dll
```

Comment:
These tests will fail even after the coreclr update won't they? As ArgIterator is not implemented. So should they be attributed to run only on Desktop framework, for now?